dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/MetricAdapter.java

```diff
@@ -77,18 +77,19 @@ public void setTags(Collection<AbstractMap.SimpleEntry<String, String>> tags) {
 
     // the constantDimensions field is only populated once, and the dimensions are reused.
     if (this.constantDimensions == null) {
```

Comment:
Please add tests for `setTags(null)` and also for calling `setTags` twice to ensure that doesn't throw.